vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -19,22 +42,16 @@
       MASTER_NODE_1: "{{ master_nodes_name.stdout_lines.1 }}"
       MASTER_NODE_2: "{{ master_nodes_name.stdout_lines.2 }}"
 
-  - set_fact:
-      WORKER_NODE_VM: "{{ WORKER_NODE | replace('-','_') }}"
-      MASTER_NODE_0_VM: "{{ MASTER_NODE_0 | replace('-','_') }}"
-      MASTER_NODE_1_VM: "{{ MASTER_NODE_1 | replace('-','_') }}"
-      MASTER_NODE_2_VM: "{{ MASTER_NODE_2 | replace('-','_') }}"
-
   - name: Get IPs of master nodes from baremetal network
     shell: |
-       kubectl get bmh "{{ MASTER_NODE_0 }}" -n metal3 -o json | jq -r '.status.hardware.nics
+       kubectl get bmh "{{ MASTER_BMH_0 }}" -n metal3 -o json | jq -r '.status.hardware.nics
        | '.[]' | select(.name|test("enp2s0")) | .ip'
-    register: MASTER_0_IP
+    register: MASTER_NODE_IP
```

Comment:
is this needed if you do not delegate the tasks ?

Reply:
see last comment

Reply:
Same here with my above comment.

Reply:
same (my above comment)

Reply:
VIP could be used here too